pkg/kubelet/nodelease/controller_test.go

```diff
@@ -195,3 +200,84 @@ func TestNewLease(t *testing.T) {
 		})
 	}
 }
+
+func TestRetryUpdateLease(t *testing.T) {
+	node := &corev1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+			UID:  types.UID("foo-uid"),
+		},
+	}
+	noConnectionUpdateErr := apierrors.NewServerTimeout(schema.GroupResource{Group: "v1", Resource: "lease"}, "put", 1)
```

Comment:
gr := schema.GroupResource{Group: "v1", Resource: "lease"}

and use it here and below (the lines will be shorter then).